### PR TITLE
Add configs

### PR DESCRIPTION
**Pull request by mlan080**
*Add configuration for database.rb

### CODE REVIEW

**Comment by mlan080** [comment]
We need both the dev and test migration in our circle CI config right?

**Comment by mlan080** [comment]
In order to run 2 migrations dev and test, we'd need 2 environment right? So I have moved my APP_ENV to the test migration and there's no longer an error. Is this the right idea? 

Also you said in the issue 'The default env should be development, then we can have a separate environment for test (which also has a different database)' , we just changed the default to test right?

File: .circleci/config.yml [hunk 1]
```diff
@@ -4,14 +4,22 @@ jobs:
...
       - checkout
       - run:
           name: bundle install
           command: bundle install
+      - run:
+          name: sequel migration test
+          command: sequel -m db/migrations sqlite://tea-shops_test.sqlite
+
+          environment:
+            APP_ENV: 'test'
```
Comment (lucasrenan) [inline .circleci/config.yml]: you should move the `environment:...` inside `docker`, then the `APP_ENV` will be available for every run.

File: .circleci/config.yml [hunk 2]
```diff
@@ -4,14 +4,22 @@ jobs:
...
+
       - run:
           name: sequel migration
-          command: sequel -m db/migrations sqlite://tea-shops.sqlite
+          command: sequel -m db/migrations sqlite://tea-shops_dev.sqlite
```
Comment (lucasrenan) [inline .circleci/config.yml]: this step should be completely removed, on CI you just need the `test` database.

**Pull request merged**